product_service_duration/views/calendar_event_view.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record id="view_calendar_event_form" model="ir.ui.view">
+        <field name="name">calendar.event.form</field>
+        <field name="model">calendar.event</field>
+        <field name="inherit_id" ref="calendar.view_calendar_event_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='alarm_ids']" position="after">
+                <field name="template_ids"
+                       widget="many2many_tags"
+                       can_create="true"
+                       can_write="true" />
+            </xpath>
+            <xpath expr="//field[@name='allday']" position="before">
+                <label for="min_duration" />
+                <div>
+                    <field name="min_duration"
+                           widget="float_time"
+                           class="oe_inline"
+                           on_change="1" />
+                    <span>
+                            hours
+                    </span>
+                </div>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="view_calendar_event_form_popup" model="ir.ui.view">
+        <field name="name">calendar.event.form</field>
+        <field name="model">calendar.event</field>
+        <field name="inherit_id" ref="calendar.view_calendar_event_form_popup" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='partner_ids']" position="after">
+                <field name="template_ids"
+                       widget="many2many_tags"
+                       can_create="true"
```

Comment:
Hmm why type products? I think earlier in the review you specified in another template ids field in the calendar popup form to do product type service. Which one should I do?